src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs

```diff
@@ -59,6 +59,7 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
             MNM.ExpressRouteCircuitSkuTier.Standard,
             MNM.ExpressRouteCircuitSkuTier.Premium,
             MNM.ExpressRouteCircuitSkuTier.Basic,
+            MNM.ExpressRouteCircuitSkuTier.Local,
```

Comment:
I forgot the history here... is Local replacing Basic? If so, what happens when the user provides Basic?

Reply:
No worries, we will support both for now will Local being the new name.